pages/dkp/kommander/2.1/major-upgrade/upgrade-clusters/migrate-platform-apps/index.md

```diff
@@ -225,13 +225,15 @@ The `kommander migrate` command requires a connection from your environment to t
     127.0.0.1   localhost localhost.localdomain <ingress_domain_name>
     ```
 
-1.  Create a TCP connection to Traefik on your cluster:
+1.  Open a separate terminal window to create a TCP connection to Traefik on your cluster:
 
     ```bash
     kubectl --kubeconfig admin.conf -n kommander port-forward svc/kommander-traefik 443:443
     ```
 
-1.  Continue with the [Move your applications](#move-your-applications) section.
+    <p class="message--note"><strong>NOTE: </strong>It is necessary that you open a new terminal window because the `kubectl port-forward` command does not return, and you need to leave this connection open for the duration of the migration.</p>
```

Comment:
Do we need to tell customers to end that connection after the migration? Or is that what happens on line 238 when we tell customers to revert the changes in the `/ect/hosts` file?

Reply:
Opening port-forward isn't a persistent change and the commend will end when terminal window is closed. Important information is already there - they need to keep it open for the duration of the migration.